### PR TITLE
Each commits adds one thing

### PR DESCRIPTION
**Pull request by zimbatm**

### CODE REVIEW

File: install-overlay [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd "$(dirname "$0")"
+
+set -x
+mkdir -p $HOME/.config/nixpkgs/overlays
+ln -sf $PWD/overlay.nix $HOME/.config/nixpkgs/overlays/nur.nix
```
Comment (Mic92) [inline install-overlay]: At the moment I have 

```nix
packageOverrides = {
  nur = callPackage (import (builtins.fetchGit {
    url = "https://github.com/nix-community/NUR";
  })) {};
};
```

in my `~/.config/nixpkgs/config.nix`
Do you think an overlay would be better?

Comment (Mic92) [reply install-overlay]: In the end we could also advertise different methods to use nur?

**Comment by Mic92** [comment]
I cherry-picked the other changes.

File: install-overlay (hunk 1 above)
Comment (zimbatm) [reply install-overlay]: Agreed, there are more possible installation methods and we should document them all. That was just the first one that came to mind.

I am not sure what the TTL for fetchGit is, does it pull on every nix evaluation?

**Comment by zimbatm** [comment]
Thanks, I'll make another PR with the doc

File: install-overlay (hunk 1 above)
Comment (Mic92) [reply install-overlay]: It updates every hour. Just like tarballs.

**Pull request closed without merging**